src/styles/theme.ts

```diff
@@ -21,6 +21,25 @@ export const common = {
     custom: {
       white: 'rgba(255, 255, 255, 1)',
     },
+    about: {
+      main: 'rgba(25, 55, 81, 1)',
+    },
+  },
```

Comment:
nope this isn't how the styles work. (ill explain in basecamp)

Reply:
revised it!